app.py

```diff
@@ -0,0 +1,66 @@
+import requests
+import bs4
+import json
+import time
+from flask import Flask
+from selenium import webdriver
+from selenium.webdriver.chrome.options import Options
+
+# chrome_options = Options()
+# chrome_options.add_argument("--headless")
+# chrome_options.add_argument("--window-size=1920x1080")
+
+app = Flask(__name__)
+@app.route("/")
+
+
+
+url = 'https://mobile.twitter.com/hashtag/'
+
+def twitterScrapper(hashtag):
+    # remove the '#' from the query
+    if '#' in hashtag :
+        hashtag = hashtag.replace('#', '')
+
+    #get the browser using selenium webdriver
+    browser = webdriver.Chrome('C:/Users/ladankhadija/SLKhadeeja/pandelytics/chromedriver.exe')
```

Comment:
instead of using the selenium driver, you could use beautiful soup to get the link in the button that says `Load older Tweets`. The use requests to make another requests

Reply:
there's no button like that. When I reach the end of the page, it just starts loading

Reply:
Turn off your javascript. You'll then see the site clearly

Reply:
I think atm, you should just return the first page. Ignore the `Load older Tweets` feature for now. Let's get something basic

Reply:
oh okay..just a moment